mmf/utils/checkpoint.py

```diff
@@ -106,8 +106,9 @@ def __init__(self, trainer):
             PathManager.mkdirs(self.models_foldername)
 
         self.save_config()
+
         self.repo_path = updir(os.path.abspath(__file__), n=3)
-        self.git_repo = None
+        self.git_repo = self.config.checkpoint.save_git_details
```

Comment:
This won't work right? This will always set self.git_repo to `true` even when `git` module is not loaded (when the repo is not git). Check it's usage in L381

Reply:
I added this option so that you can override saving of git details even in the case of git repo.

Reply:
But won't this crash when `import git` fails?

Reply:
You are right fixed this.